src/core/iomux/io_mux_call.cpp

```diff
@@ -252,10 +252,8 @@ bool io_mux_call::handle_os_countdown(int &poll_os_countdown)
      */
     if (poll_os_countdown-- <= 0 && m_n_sysvar_select_poll_os_ratio > 0) {
         if (wait_os(true)) {
-            // This will empty the cqepfd
-            // (most likely in case of a wakeup and probably only under epoll_wait (Not
-            // select/poll))
-            ring_clear_rx_notification();
+            // DOCA WA: Unclearable manual triggered events generate false PE events.
```

Comment:
[suggestion] Put "XXX" or "TODO" prefix. Some editors such as vim highlight it and this will raise attention to a temporary code.

Reply:
Done